scripts/ASTImportTest.sh

```diff
@@ -169,7 +305,20 @@ function test_import_export_equivalence {
         # and print some details about the corresponding solc invocation.
         if (( solc_return_code == 2 ))
         then
-            fail "\n\nERROR: Uncaught Exception while executing '$SOLC ${compile_test} ${input_files[*]}':\n${output}\n"
+            # For the evm-assembly import/export tests, this script uses only the
+            # old code generator. Some semantic test can only be compiled with
+            # --via-ir (some need to be additionally compiled with --optimize).
+            # The tests that are meant to be compiled with --via-ir are throwing
```

Comment:
Actually, we should probably test it with `--via-ir` as well. I mean, when exporting the initial asm JSON that you later import. That JSON will be different than on legacy and might uncover some corner cases that we do not hit there.

Reply:
Thats true. It is always good to use as many inputs as possible for testing.

Reply:
Marking this as resolved since in the end it turned out that the `--via-ir` test are way too slow (50+ min) to be worth it and we had to remove them.